ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/worldstate/DefaultMutableWorldState.java

```diff
@@ -49,7 +49,7 @@
   private final WorldStateStorage worldStateStorage;
 
   public DefaultMutableWorldState(final WorldStateStorage storage) {
-    this(MerklePatriciaTrie.EMPTY_TRIE_ROOT_HASH, storage);
+    this(MerklePatriciaTrie.EMPTY_TRIE_NODE_HASH, storage);
```

Comment:
There's nothing special about a root node, so updated this constant name so that it can be used more generally.